src/dialects/postgres/index.js

```diff
@@ -63,6 +63,9 @@ export class PostgresDialect extends AbstractDialect {
     this.queryGenerator = new PostgresQueryGenerator({
       dialect: this,
       sequelize,
+      options: {
```

Comment:
QueryGenerator isn't responsible for the creation of Query objects, I don't think this option bag will reach it

The query object is created by `Sequelize#queryRaw`:

```
const query = new this.dialect.Query(connection, this, options);
```

IMO the `logWarnings` could get the dialect name from `this.sequelize.dialect.name`. It would be fully lowercase but I think that's fine, that's already what our other messages do.

Reply:
@ephys `lodash.capitalize` could improve `this.sequelize.dialect.name` :)

Reply:
I think we should consider styling the dialect name directly so `capitalize` is not needed. So we could use both `MariaDB`/`MySQL` and `mariadb`/`mysql`. The first one for displaying it to users, the second one internally

Reply:
hola saludos desde ecuador, usamos sequelize en nuestra empresa :)

Reply:
We'll add a stylized name of dialect in a future refactor. For now the lowercase one is fine